diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
index c24e47ac3f0e0bc7aa8d9a68a5f0a6b3dd782480..d52245570884d23d502c4cd1b1aa70946ee445e5 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bubble_view.cc
@@ -375,6 +375,7 @@ void BookmarkBubbleView::ShowBubble(
                                        bookmark_node);
 
   bool show_simplified_flow = !already_bookmarked;
+  BRAVE_BOOKMARK_BUBBLE_VIEW_SHOW_BUBBLE_SHOW_SIMPLIFIED
 
   auto bubble_delegate_unique = std::make_unique<BookmarkBubbleDelegate>(
       std::move(delegate), browser, url, show_simplified_flow);
@@ -516,6 +517,7 @@ void BookmarkBubbleView::ShowBubble(
   auto bubble = std::make_unique<views::BubbleDialogModelHost>(
       std::move(dialog_model), anchor_view, views::BubbleBorder::TOP_RIGHT);
   bookmark_bubble_ = bubble.get();
+  BRAVE_BOOKMARK_BUBBLE_VIEW_SHOW_BUBBLE_SET_ARROW
   if (highlighted_button)
     bubble->SetHighlightedButton(highlighted_button);
 
