diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 444578446933d08edac6e5c4aa51ebc275834729..4255d560fc2e27d2dbaddf5e2aed112dcf606dfa 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2767,6 +2767,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
