diff --git a/chrome/browser/ui/startup/startup_browser_creator_impl.cc b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
index 497ba7b735c47ccdc47ea3c85baf236fd9607d91..c3c95c568f2e4ccc27d952a879cc4ab005ab5383 100644
--- a/chrome/browser/ui/startup/startup_browser_creator_impl.cc
+++ b/chrome/browser/ui/startup/startup_browser_creator_impl.cc
@@ -524,6 +524,7 @@ StartupBrowserCreatorImpl::DetermineStartupTabs(
       }
 #endif
 
+      BRAVE_STARTUPBROWSERCREATORIMPL_DETERMINEURLSANDLAUNCH
       // Potentially add the What's New Page. Note that the What's New page
       // should never be shown in the same session as any first-run onboarding
       // tabs. It also shouldn't be shown with reset tabs that are required to
