diff --git a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
index 74f1129d7b72f72b698f687ef905c05689fb943e..c139e5bdd78d49fb7cd2ea53410f5e1fe3aeab8c 100644
--- a/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
+++ b/third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc
@@ -302,7 +302,7 @@ int GetMicrotasksScopeDepth(v8::Isolate* isolate,
 
 }  // namespace
 
-v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
+v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript_ChromiumImpl(
     ScriptState* script_state,
     const ClassicScript& classic_script,
     v8::ScriptOrigin origin,
@@ -346,7 +346,7 @@ v8::MaybeLocal<v8::Script> V8ScriptRunner::CompileScript(
   return script;
 }
 
-v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule(
+v8::MaybeLocal<v8::Module> V8ScriptRunner::CompileModule_ChromiumImpl(
     v8::Isolate* isolate,
     const ModuleScriptCreationParams& params,
     const TextPosition& start_position,
